lib/zmq-kernel.coffee

```diff
@@ -306,30 +320,27 @@ class ZMQKernel extends Kernel
 
 
     destroy: ->
-        super
-        console.log 'sending shutdown'
+        console.log 'ZMQKernel: destroy:', this
 
-        message = @_createMessage 'shutdown_request'
-        message.content =
-            restart: false
-
-        @shellSocket.send new jmp.Message message
+        if @kernelProcess?
+            @interrupt()
```

Comment:
Good catch, `SIGINT` is for interrupting the kernel and `SIGKILL` is to kill it.